devices/danfoss.js

```diff
@@ -78,7 +79,18 @@ module.exports = [
                 .withDescription('Mean radiator load for room calculated by gateway for load balancing purposes (-8000=undefined)')
                 .withValueMin(-8000).withValueMax(2000),
             exposes.numeric('load_estimate', ea.STATE_GET)
-                .withDescription('Load estimate on this radiator')],
+                .withDescription('Load estimate on this radiator'),
+            exposes.binary('preheat_status', ea.STATE_GET, true, false)
+                .withDescription('Specific for pre-heat running in Zigbee Weekly Schedule mode'),
+            exposes.enum('adaptation_run_status', ea.STATE_GET, ['None', 'In progress', 'Found', 'Lost'])
+                .withDescription('bit0=Adaptation run in progress, bit1=Valve Characteristic found, bit2=Valve Characteristic lost'),
+            exposes.binary('adaptation_run_settings', ea.ALL, true, false)
+                .withDescription('Automatic adaptation run enabled (the one during the night)'),
+            exposes.enum('adaptation_run_control', ea.ALL, ['None', 'Initate Adaption', 'Cancel Adaption'])
```

Comment:
The values here should match the ones from `danfossAdaptationRunControl` (e.g. `initate_adaptation`)